packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -120,6 +126,8 @@ describe('plugin-meetings', () => {
 
       // now simulate server requiring us to locally unmute
       audio.handleServerLocalUnmuteRequired(meeting);
+      meeting.mediaProperties.audioStream.setServerMuted(false,'localUnmuteRequired');
```

Comment:
Sorry. I've forgotten to remove this while cleaning up. I should have added that to test something and missed to remove. It was before I understood it was my stub mistake in muteState and not because we flipped the default mute value. Thanks for noticing it.

Reply:
Removed this now